extdata/InlineComments.txt

```diff
@@ -10,5 +10,5 @@
 .cobra : """ """
 .f .for .f90 .f95 .f03 .factor : ! {}
 .json : {} {}
-.pi .erl .ebin .escript .fp : % {}
+.erl .ebin .escript .fp .pi .pro : % {}
```

Comment:
use .pl instead of .pi

Reply:
I didn't add .pi, I only moved it to sort the line in alphabetic order.

Reply:
Ok, my bad.